cdap-app-fabric/src/main/java/io/cdap/cdap/app/store/Store.java

```diff
@@ -387,13 +387,13 @@ List<ProgramSpecification> getDeletedProgramSpecifications(ApplicationReference
   ApplicationMeta getLatest(ApplicationReference appRef);
 
   /**
-   * Scans for applications across all namespaces.
+   * Scans for the latest applications across all namespaces.
    *
    * @param txBatchSize maximum number of applications to scan in one transaction to
    *                    prevent holding a single transaction for too long
    * @param consumer a {@link BiConsumer} to consume each application being scanned
    */
-  void scanApplications(int txBatchSize, BiConsumer<ApplicationId, ApplicationMeta> consumer);
+  void scanLatestApplications(int txBatchSize, BiConsumer<ApplicationId, ApplicationMeta> consumer);
```

Comment:
I don't think we need to say latest in the name. Application  is always latest. The application version can be latest or not

Reply:
Sounds good, reverted the name change